src/lib/merkle_address/merkle_address.mli

```diff
@@ -0,0 +1,76 @@
+open Core_kernel
+
+type t [@@deriving sexp, hash, eq, compare, to_yojson]
+
+module Stable : sig
```

Comment:
can use `%%versioned:`, I think

Reply:
`%%versioned:` creates an unsatisfiable module equality if we use it here. I'll put updating the ppx in my backlog, but I think we're stuck with this for now :/